assignment1.txt

```diff
@@ -0,0 +1,15 @@
+Harry Brickner
+
+1) It isn't required, but it's probably considered bad etiquette to leave it out. If it's anything like java's System.exit, the number returned should probably have something to do with the reason that the program is exiting. 0 means it is ending willingly, whereas other numbers might correspond to error messages.
+
+2) X is a float, but it is being printed as an integer. This shouldn't throw any errors (as far as I know) but the message claims that it is printing the value of x rather than the rounded value... Also, it's missing the return 0, which isn't necessary but it's bad etiquette to leave it out, probably
```

Comment:
yes - the return 0; should be included! As far as the %d vs %f, it should be %f since you didn't cast (int) around the x. Lastly, you're missing one more error - see if you can find it.